ch_backup/clickhouse/control.py

```diff
@@ -312,15 +312,10 @@ def __init__(self, ch_ctl_config: dict, main_config: dict) -> None:
             "allow_experimental_funnel_functions": 1,
             "allow_experimental_live_view": 1,
             "allow_experimental_window_view": 1,
+            "allow_experimental_object_type": 1,
             "allow_suspicious_codecs": 1,
             "allow_suspicious_low_cardinality_types": 1,
         }
-        if self.ch_version_ge("22.3"):
-            settings.update(
-                {
-                    "allow_experimental_object_type": 1,
-                }
-            )
```

Comment:
22.3 is a minimal supporting version. So it's safe to set `allow_experimental_object_type` unconditionally.